index.html

```diff
@@ -4377,18 +4383,75 @@ <h3>Top level <code>forms</code></h3>
     ...
     "forms": [{
         "op": "readallproperties",
-        "href": "https://mylamp.example.com/allproperties",
+        "href": "https://mylamp.example.com/properties",
         "contentType": "application/json",
         "htv:methodName": "GET"
-    }, {
+    }, 
+    {
         "op": "writeallproperties",
-        "href": "https://mylamp.example.com/allproperties",
+        "href": "https://mylamp.example.com/properties",
         "contentType": "application/json",
         "htv:methodName": "PUT"
     }]
 }
 </pre>
 
+<p>
+Thing-level <code>uriVariables</code> can be used here to supply further variables to the operation or to specify 
+a list of Property Affordance names for a <code>readmultipleproperties</code> operation. 
+In the example below, the unit for the properties can be set via such a variable and the desired list of properties 
+can be set: 
+</p>
+
+<pre class="example" id="td-forms-urivariables-thing-example">
+    {
+        ...
+        "properties": {
+            "temperature": {
+                "type": "number",
+                "forms": [...]
+            },
+            "brightness": {
+                "type": "number",
+                "forms": [...]
+            },
+            "humidity": {
+                "type": "integer",
+                "forms": [...]
+            }
+        },
+        "uriVariables":{
+            "propertyNames":{
+                "type": "string",
+                "description": "Comma separated list of property names to select."
+            },
+            "unitSystem":{
+                "type": "string",
+                "enum":["metric","imperial","uscustomary"],
+                "description":"System of Measurement that will be used for the values"
+            }
+        }
+        "forms": [{
+            "op": "readallproperties",
+            "href": "https://mything.example.com/properties{?unitSystem}",
+            "contentType": "application/json",
+            "htv:methodName": "GET"
+        }, 
+        {
+            "op": "readmultipleproperties",
+            "href": "https://mylamp.example.com/properties{?fields,unitSystem}",
```

Comment:
You defined the uriVariable `propertyNames` above, but referenced `fields` here in the IRI.